packages/macro/src/index.macro.ts

```diff
@@ -63,33 +96,31 @@ const macro: MacroHandler = ({ references, state }) => {
   }
   const side : 'client' | 'server' = opts?.side;
 
-  // const contentsCache: { [filename: string]: string } = {
-  //   "/": state.file.code,
-  // };
-
-  // const readFile = (filename: string): string => {
-  //   let contents = contentsCache[filename];
-  //   if (contents == null) {
-  //     contents = fs.readFileSync(filename, "utf-8");
-  //     contentsCache[filename] = contents;
-  //   }
-
-  //   return contents;
-  // };
-
   (references.useSSRComputation || []).map((nodePath: NodePath) => {
     const parent = nodePath.parent;
     if (t.isCallExpression(parent)) {
-      if (parent.arguments.length < 1) { 
-        throw new Error("useSSRComputation must be called with at least one arguments: a path to a .ssr-computation.js file containing the definition of the funciton.");
+      if (parent.arguments.length < 2) { 
+        throw new Error("useSSRComputation must be called with at least two arguments: a path to a .ssr-computation.js file containing the definition of the funciton and array of dependencies.");
+      }
+
+      if (parent.arguments.length > 3) {
+        throw new Error("useSSRComputation must be called with at most three arguments: a path to a .ssr-computation.js file containing the definition of the funciton, array of dependencies and options object.");
```

Comment:
Should I limit the number of parameters passed?

Reply:
Yeah, this looks good.

Also, update typescript declaration:

https://github.com/Popmenu/use-ssr-computation.macro/pull/29/files#diff-79dbaa8b7fc8790d4fbb7957e106b63f9335fd377a9f2ba106766e89613c6bceR176

Reply:
Done